functions/register/main.py

```diff
@@ -101,18 +102,10 @@ def _is_request_valid(request: Request) -> Tuple[bool, Optional[Tuple[Response,
 
 
 def _check_phone_number(msisdn: str):
-    msisdn = msisdn.strip().replace(" ", "")
-    if not msisdn.startswith("+48"):
-        logging.warning(f"check_phone_number: invalid prefix: {msisdn}")
-        return False
-    if len(msisdn) != 12:
-        logging.warning(f"check_phone_number: invalid msisdn length: {msisdn}")
-        return False
-
-    try:
-        int(msisdn)
-    except ValueError:
-        logging.warning(f"check_phone_number: invalid value: {msisdn}")
+    msisdn = re.sub('[^0-9,+]', '', msisdn)
+    if (not msisdn.startswith("+48") and len(msisdn) != 12) \
```

Comment:
Since `re` is here, why not use another regexp to validate?
```regex
^\+?[0-9]{11}$
```
https://regex101.com/r/tlKw5U/2

Reply:
You version is much cleaner, but probably a bit slower with bigger load (len is O(1) and startswith constant). I'm fine with both versions.

Reply:
You call `re.sub` above so it shouldn't change complexity overall.